packages/electron/template/main/index.js

```diff
@@ -18,15 +18,19 @@ function createWindow () {
 		height: 600,
 		icon: (process.platform==='win32'
 				? path.join(__dirname, 'assets', 'icon-mini.png')
-				: path.join(__dirname, 'assets', 'icon.png'))
+				: path.join(__dirname, 'assets', 'icon.png')),
+		webPreferences: {
+			nodeIntegration: true,
+			contextIsolation: false,
+		}
```

Comment:
this was added in order to make the app start successfuly

Reply:
🤔 I'm still seeing the "global is not defined". Let's take a look at this in another ticket.